ci-operator/step-registry/hypershift/mce/kubevirt/destroy/hypershift-mce-kubevirt-destroy-chain.yaml

```diff
@@ -10,23 +10,30 @@ chain:
           source "${SHARED_DIR}/proxy-conf.sh"
         fi
 
+        MCE_VERSION=$(oc get "$(oc get multiclusterengines -oname)" -ojsonpath="{.status.currentVersion}" | cut -c 1-3)
+        HYPERSHIFT_NAME=hcp
+        if (( $(echo "$MCE_VERSION < 2.4" | bc -l) )); then
+          echo "MCE version is less than 2.4, use hypershift command"
+          HYPERSHIFT_NAME=hypershift
+        fi
+
         arch=$(arch)
         if [ "$arch" == "x86_64" ]; then
-          downURL=$(oc get ConsoleCLIDownload hypershift-cli-download -o json | jq -r '.spec.links[] | select(.text | test("Linux for x86_64")).href') && curl -k --output /tmp/hypershift.tar.gz ${downURL}
-          cd /tmp && tar -xvf /tmp/hypershift.tar.gz
-          chmod +x /tmp/hypershift
+          downURL=$(oc get ConsoleCLIDownload ${HYPERSHIFT_NAME}-cli-download -o json | jq -r '.spec.links[] | select(.text | test("Linux for x86_64")).href') && curl -k --output /tmp/${HYPERSHIFT_NAME}.tar.gz ${downURL}
+          cd /tmp && tar -xvf /tmp/${HYPERSHIFT_NAME}.tar.gz
+          chmod +x /tmp/${HYPERSHIFT_NAME}
           cd -
         fi
 
         CLUSTER_NAME="$(echo -n $PROW_JOB_ID|sha256sum|cut -c-20)"
         
-        /tmp/hypershift dump cluster --artifact-dir=$ARTIFACT_DIR \
+        /tmp/${HYPERSHIFT_NAME} dump cluster --artifact-dir=$ARTIFACT_DIR \
```

Comment:
just like `hypershift install` is only available in the `hypershift` binary and not `hcp. The `hypershift dump` command is also not available in the `hcp` binary. 

For the dump command, use the `hypershift` cli. otherwise we won't have test assets to look at when failures occur.